### PR TITLE
Add new models for membership service

### PR DESCRIPTION
**Pull request by abutterf**

### CODE REVIEW

File: lib/ims/lti/models/serializable.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module IMS::LTI::Models
+  module Serializable
+    def as_json
```
Comment (rivernate) [inline lib/ims/lti/models/serializable.rb]: active support expects as_json to be able to take a options hash as an argument, we just add it to keep error from being thrown.  see: http://api.rubyonrails.org/classes/ActiveModel/Serializers/JSON.html

File: lib/ims/lti/models.rb [hunk 2]
```diff
@@ -31,5 +31,10 @@ module Models
     require_relative 'models/content_item_placement'
     require_relative 'models/content_item_container'
     require_relative 'models/image'
+    require_relative 'models/agent'
```
Comment (rivernate) [inline lib/ims/lti/models.rb]: we should probably namespace the membership service models in the off chance IMs creates conflicting models in the future

**Comment by rivernate** [comment]
we might want to look at using this: https://github.com/rails-api/active_model_serializers/blob/master/docs/general/serializers.md

**Comment by defektive** [comment]
:+1:

**Pull request merged**